src/uml-builder.ts

```diff
@@ -32,8 +34,24 @@ export function buildUml(modules: Module[], outputFilename: string, dependencies
         }
     }
 
+    const configFile = process.cwd() +  "/tsviz-config.json";
```

Comment:
Move the configuration read to the main and make it configurable ... ie allow specifying alternative path